content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+# Sourcegraph 4.0
```

Comment:
The template for the 4.0 release post will be as follows and it can be found in this doc: https://docs.google.com/document/d/1Ng5pzuCH80DavqXlp8PxAM8DOaqFl7o8m_puzVCAuAs/edit?usp=sharing

Subhead: The code intelligence platform for the modern development team

Intro: TODO

Theme 1: Code intelligence: more useful, more times daily
TEMPLATE 
The information below should be provided by PMs for all new features that will be included in Sourcegraph 4.0

Headline: [Do X] with [Z]
Description: 2 - 4 sentences on the feature, functionality, and benefits
Media: provide a screenshot, gif, or video to help support the copy
Tag by product: [Code Search], [Batch Changes], [Code Navigation], [Code Insights], [Admin]
Documentation: provide a link to the documentation

Theme 2: High-leverage ways to improve your codebase
TEMPLATE 
The information below should be provided by PMs for all new features that will be included in Sourcegraph 4.0

Headline: [Do X] with [Z]
Description: 2 - 4 sentences on the feature, functionality, and benefits
Media: provide a screenshot, gif, or video to help support the copy
Tag by product: [Code Search], [Batch Changes], [Code Navigation], [Code Insights], [Admin]
Documentation: provide a link to the documentation

Theme 3: Love for admins 

TEMPLATE 
The information below should be provided by PMs for all new features that will be included in Sourcegraph 4.0

Headline: [Do X] with [Z]
Description: 2 - 4 sentences on the feature, functionality, and benefits
Media: provide a screenshot, gif, or video to help support the copy
Tag by product: [Code Search], [Batch Changes], [Code Navigation], [Code Insights], [Admin]
Documentation: provide a link to the documentation

Theme 4: Dedicated enterprise cloud deployment
TEMPLATE 
The information below should be provided by PMs for all new features that will be included in Sourcegraph 4.0

Headline: [Do X] with [Z]
Description: 2 - 4 sentences on the feature, functionality, and benefits
Media: provide a screenshot, gif, or video to help support the copy
Tag by product: [Code Search], [Batch Changes], [Code Navigation], [Code Insights], [Admin]
Documentation: provide a link to the documentation


To the PMs - please provide your inputs to this PR by EOD 8/31 in the format listed above.

@benvenker @malomarrec @Joelkw @macraig @efritz @ryphil @ryankscott @quinnhare @taylorsperry

Reply:
@lcolston08 @iskyOS is it possible for marketing to make the PR to first input the template as you would like it? It seems like this is just a format change, rather than an information change. (We all have the information on our features in this post already.) 

There's a chance we'll end up with some odd formatting inconsistencies if we all patchwork various aspects of this template and it'll probably be much faster to have 1 person take the information already in this blog post rather than have to chase down 9 other individuals to just make the same reformatting change (if I understand that's what this request is).